api/tests/unit/domain/usecases/create-certification-center-membership-by-email_test.js

```diff
@@ -0,0 +1,81 @@
+const { catchErr, expect, sinon } = require('../../../test-helper');
+
+const {
+  AlreadyExistingEntityError,
+  UserNotFoundError,
+} = require('../../../../lib/domain/errors');
+
+const createCertificationCenterMembershipByEmail = require('../../../../lib/domain/usecases/create-certification-center-membership-by-email');
+
+describe('Unit | UseCase | create-certification-center-membership-by-email', () => {
+
+  const certificationCenterId = 1;
+  const email = 'user@exemple.net';
+  const userId = 1;
+
+  let certificationCenterMembershipRepository;
+  let userRepository;
+
+  beforeEach(() => {
+    certificationCenterMembershipRepository = {
+      isMemberOfCertificationCenter: sinon.stub(),
+      save: sinon.stub(),
+    };
+    userRepository = {
+      getByEmail: sinon.stub(),
+    };
+
+    certificationCenterMembershipRepository.isMemberOfCertificationCenter.resolves(false);
+    certificationCenterMembershipRepository.save.resolves();
+    userRepository.getByEmail.resolves({ id: userId });
+  });
+
+  it('should save certification center membership', async () => {
```

Comment:
**problème**
Si je lis le nom du test, je me rends compte qu'il ne fait pas ce qu'il dit, donc je propose
- renommer en `should call repositories`
- séparer en 3 tests, ex` should retrieve user` pour `expect(userRepository.getByEmail).has.been.calledWith(email);`

Reply:
Done : renommer en `should call repositories`